tests/test_other.py

```diff
@@ -7952,16 +7940,18 @@ def test(filename, expectations):
         (['-O3'],  0, [],         [],          55,  0,  1, 1), # noqa
         (['-Os'],  0, [],         [],          55,  0,  1, 1), # noqa
         (['-Oz'],  0, [],         [],          55,  0,  1, 1), # noqa
-      ])
+      ], size_slack_factor)
 
       print('test on libc++: see effects of emulated function pointers')
       test(path_from_root('tests', 'hello_libcxx.cpp'), [
         (['-O2'], 36, ['assert'], ['waka'], 196709,  30,   41, 659), # noqa
         (['-O2', '-s', 'EMULATED_FUNCTION_POINTERS=1'],
                   36, ['assert'], ['waka'], 196709,  30,   22, 620), # noqa
-      ]) # noqa
+      ], size_slack_factor) # noqa
     else:
       # wasm-backend
+      size_slack_factor = 0.50  # changes quite a bit
```

Comment:
Can you explain a little more?   Does it change form run-to-run?  Or do you you mean the upstream change tend to make this vary from day to day?   Does this mean it can vary up to 50%?  That seems like a lot of variability.  I'm probably missing something :)

Reply:
Not run to run, but the anticipated irreducible control flow optimization is a big change, and in general there is upstream churn here, that I've already noticed in just a few weeks. I'll clarify the comment, but the idea is "mostly ignore code size here for now".